src/EditorFeatures/Core/Implementation/CodeFixes/CodeFixService.cs

```diff
@@ -365,6 +389,34 @@ public async Task<Document> ApplyCodeFixesForSpecificDiagnosticIdAsync(Document
             var task = fixer.RegisterCodeFixesAsync(context) ?? Task.CompletedTask;
             await task.ConfigureAwait(false);
             return fixes.ToImmutable();
+
+            static ImmutableArray<Diagnostic> FilterApplicableDiagnostics(
+                ImmutableArray<Diagnostic> applicableDiagnostics,
+                string equivalenceKey,
+                Dictionary<Diagnostic, PooledHashSet<string>> uniqueDiagosticToEquivalenceKeysMap)
+            {
+                using var disposer = ArrayBuilder<Diagnostic>.GetInstance(out var newApplicableDiagnostics);
```

Comment:
```suggestion
                using var _ = ArrayBuilder<Diagnostic>.GetInstance(out var newApplicableDiagnostics);
```

Reply:
Thanks, will do this as part of a follow-up change.